src/cli/submit/SubmitJobHandler.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { JobUtils } from "../../api/JobUtils";
+import { FTPBaseHandler } from "../../FTPBase.Handler";
+import { IFTPHandlerParams } from "../../IFTPHandlerParams";
+
+const ONE_SECOND = 1000;
+const DEFAULT_INTERVAL = 5000;
+const DEFAULT_MAX_TRIES = 12;
+
+export abstract class SubmitJobHandler extends FTPBaseHandler {
```

Comment:
Having a Common/Base handler is nice. However, shouldn't most of this logic be contained in the Plugin "SDK" (or the TS APIs of the plugin, i.e. `api` folder and exported as such)?
That way it's easier for the https://github.com/zowe/zowe-explorer-ftp-extension/ to consume instead of having to workaround handlers 😋

Reply:
This will not be considered in this PR but we plan to support the ftp extention in the future. 😊

Reply:
Created https://github.com/zowe/zowe-cli-ftp-plugin/issues/57
Thanks!